wallets/provider-trezor/src/helpers.ts

```diff
@@ -0,0 +1,54 @@
+import { cleanEvmError } from '@rango-dev/signer-evm';
+import { ETHEREUM_CHAIN_ID, Networks } from '@rango-dev/wallets-shared';
+
+export const ETHEREUM_BIP32_PATH = "m/44'/60'/0'/0/0";
+
+export function getTrezorErrorMessage(error: any) {
+  if (error?.shortMessage) {
+    return new Error(error?.shortMessage);
```

Comment:
To have a rich error reporting please add `cause` as well. (`, {cause: error}`)
Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause

Reply:
Please also add a comment why you are using `shortMessage`, if there is any doc on Trezor, you can add the link

Reply:
Same comment applies to other places that you are re-throwing errors. (e.g. `getEthereumAccounts` function)

Reply:
I fixed it and added a comment. I couldn't find any doc on Trezor.